ktor-client/ktor-client-features/ktor-client-json/ktor-client-serialization/js/src/SerializerInitializer.kt

```diff
@@ -0,0 +1,10 @@
+import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
+
+val initializer = SerializerInitializer
```

Comment:
Same here: one should't use initializer and `SerializerInitializer`. I'd prefer to hide/mark it somehow.